### ISSUE TITLE
Other actions

### ISSUE THREAD

**Issue by rayman89** [koluch/tabs-keeper#2]
I was thinking if you were planing into making this addon like a tab manager addon it would be nice to have like a right click menu that allows you to do other stuff to the tab like reload the tab, bookmark it, move it to an other window, unload the tab, copy it's url etc

**Comment by koluch** [comment]
Yes, I am going to support more operations you can do with a tab. I don't think that I will bind it to right mouse button, but it will be some kind of context menu.
